### ISSUE TITLE
[Link and button labels] Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s. (08.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30019]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Ryan Thurlwell (OCTO Design Lead)

---

### Platform Issue
Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s.

### Issue Details
The are 2 different buttons for adding and removing dependents. The modal has Change dependents. The rest of the site has Add or remove dependents.

### VA.gov Experience Standard
[Category Number 08, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.6_AA

---

### Platform Recommendation
The button "Change dependents" should be "Add or remove dependents" to match the other buttons

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved
